cli/app/app.go

```diff
@@ -1165,9 +1182,9 @@ func Convert(c *cli.Context) {
 func checkUnsupportedKey(service interface{}) {
 	s := structs.New(service)
 	for _, f := range s.Fields() {
-		if f.IsExported() && !f.IsZero() {
+		if f.IsExported() && !f.IsZero() && strings.Compare(f.Name(), "Networks") != 0 {
```

Comment:
what is the main benefit of replacing `strings.Compare()` with `!=` to compare strings?